query-engine/connectors/sql-query-connector/src/row.rs

```diff
@@ -246,7 +246,7 @@ pub fn row_value_to_prisma_value(p_value: Value, type_identifier: &TypeIdentifie
                 return Err(SqlError::ConversionError(error.into()));
             }
         },
-        TypeIdentifier::Int => match p_value {
+        TypeIdentifier::Int | TypeIdentifier::BigInt => match p_value {
```

Comment:
Why don't we need to create a `PrismaValue::BigInt(x)` here?

Reply:
Because of type coercions on the higher levels.